HIP/hip-xxx.md

```diff
@@ -0,0 +1,921 @@
+                "0000000000000000000000000000000000000000000000000000000000000014",
+                "0000000000000000000000000000000000000000000000000000000000000014"
+            ],
+            "memory": [
+                "0x0000000000000000000000000000000000000000000000000000000000000000",
+                "0000000000000000000000000000000000000000000000000000000000000000",
+                "0000000000000000000000000000000000000000000000000000000000000080"
+            ],
+            "storage": {
+                "0x0000000000000000000000000000000000000000000000000000000000000000": "0x0000000000000000000000000000000000000000000000000000000000000014"
+            },
+            "reason": null
+        },
+        {
+            "pc": 546,
+            "op": "SLOAD",
+            "gas": 4411,
+            "gasCost": 2100,
+            "depth": 1,
+            "stack": [
+                "0x00000000000000000000000000000000000000000000000000000000094f6253",
+                "00000000000000000000000000000000000000000000000000000000000000a5",
+                "0000000000000000000000008fa74f0a79112c8c46c8e6739eb8cc929083298c",
+                "0000000000000000000000000000000000000000000000000000000000000014",
+                "0000000000000000000000000000000000000000000000000000000000000014",
+                "0000000000000000000000000000000000000000000000000000000000000001"
+            ],
+            "memory": [
+                "0x0000000000000000000000000000000000000000000000000000000000000000",
+                "0000000000000000000000000000000000000000000000000000000000000000",
+                "0000000000000000000000000000000000000000000000000000000000000080"
+            ],
+            "storage": {
+                "0x0000000000000000000000000000000000000000000000000000000000000000": "0x0000000000000000000000000000000000000000000000000000000000000014"
+            },
+            "reason": null
+        },
+        [...]
+    ]
+}    
+```
+
+**Output**: detailed information for the transaction execution with `stack` set to `false`
+Example output:
+
+```
+{
+        "gas": 24116,
+        "failed": false,
+        "returnValue": "",
+        "structLogs": [
+            [...],
+            {
+                "pc": 541,
+                "op": "SLOAD",
+                "gas": 4519,
+                "gasCost": 100,
+                "depth": 1,
+                "stack": null,
+                "memory": [
+                    "0x0000000000000000000000000000000000000000000000000000000000000000",
+                    "0000000000000000000000000000000000000000000000000000000000000000",
+                    "0000000000000000000000000000000000000000000000000000000000000080"
+                ],
+                "storage": {
+                    "0x0000000000000000000000000000000000000000000000000000000000000000": "0x0000000000000000000000000000000000000000000000000000000000000014"
```

Comment:
Worth considering if it's worthwhile performance wise to remove leading zeros for the REST API response. Though it would mean the relay would have to re-add them I believe to be spec conformant?

Reply:
Yes, if we remove the leading zeros, the relay should populate them. We would gain performance benefits on the mirror-node side, but the overall performance might be the same, due to the extra computation on the relay side.

Reply:
I don't think it'd scale.
The example likely includes slots that relate to simple data type variables but not all will be and the majority within a class may not have leading zeroes